changelog.txt

```diff
@@ -5,6 +5,7 @@
 
 = 2.2.0 - 2023-01-19 =
 * Add   - Add option to let user pick whether to save the last package & service or not.
+* Fix   - Adjust checkout US zipcode validation to run only when exactly 5 or 10 digits are typed.
```

Comment:
@dustinparker I saw this commit merged but we already released `2.2.0`. Do we need to release 2.2.2. including this PR? If so, we will need to change this changelog (and readme).

Reply:
I have already released this ([diff](https://github.com/Automattic/woocommerce-services/compare/aaedea0ac6c67616adfc3c651b91dc0cfd192316...2.2.2)) by moving the changelog entry but it would be great if you could please confirm if this 👆 is correct.

Reply:
Sorry guys, I didn't get this notification for some reason until today. Yes, this is just a fix so 2.2.2 is fine.